src/next-collection-view.js

```diff
@@ -577,6 +577,17 @@ const CollectionView = Backbone.View.extend({
     this.Dom.appendContents(this.el, els, {_$el: this.$el});
   },
 
+  swapChildViews(view1, view2) {
+    if (!this.children.hasView(view1) || !this.children.hasView(view1)) {
```

Comment:
I'm ok with the feature but allowing to swap between to different collection views will complicate the things